.gitignore

```diff
@@ -8,6 +8,7 @@
 /ports/android/obj
 /python/_virtualenv
 /python/tidy/servo_tidy.egg-info
+/etc/ci/performance/page_load_test
```

Comment:
How about adding this to the `etc/ci/performance/.gitignore` instead? If someday we want to move the `etc/ci/performance` folder we don't need to remember to edit this top level gitignore.